### PR TITLE
Migrate `src/utils/jwt` to TypeScript

### PR DESCRIPTION
**Pull request by ChrisAntaki**

### CODE REVIEW

File: src/utils/jwt.ts [hunk 1]
```diff
@@ -35,19 +32,12 @@ export class JwtHelper {
 
   /**
    * Decodes JWT token and returns its payload.
-   * @param {string} encodedToken
-   * @return {?JsonObject|undefined}
    */
-  decode(encodedToken) {
+  decode(encodedToken: string): any {
```
Comment (fikus) [inline src/utils/jwt.ts:36]: We should minimize use of `any`, especially in public APIs. If this is intended to be opaque, can you update the documentation to say so and change the return type to `{}`?

File: src/utils/jwt.ts [hunk 2]
```diff
@@ -35,19 +32,12 @@ export class JwtHelper {
...
     return this.decodeInternal_(encodedToken).payload;
   }
 
-  /**
-   * @param {string} encodedToken
-   * @return {!JwtTokenInternalDef}
-   * @private
-   */
-  decodeInternal_(encodedToken) {
+  decodeInternal_(encodedToken: string): JwtTokenInternalDef {
```
Comment (fikus) [inline src/utils/jwt.ts:40]: Should be marked `private`.

File: src/utils/jwt.ts (hunk 1 above)
Comment (ChrisAntaki) [reply src/utils/jwt.ts:36]: Updated!

File: src/utils/jwt.ts (hunk 2 above)
Comment (ChrisAntaki) [reply src/utils/jwt.ts]: Updated!

File: src/utils/jwt.ts (hunk 1 above)
Comment (ChrisAntaki) [reply src/utils/jwt.ts:36]: In this case, I think `any` is appropriate since a JWT could have anything in its payload

Comment (ChrisAntaki) [reply src/utils/jwt.ts:36]: As a general utility, this needs to support a variety of JWTs. Perhaps the code calling this method could cast the value to a more specific type?

**Review by fikus — APPROVED** [review]

File: src/utils/jwt.ts (hunk 1 above)
Comment (ChrisAntaki) [reply src/utils/jwt.ts:36]: Cool, I think `any` works well here. When we migrate JavaScript files that expects a certain kind of JSON in the payload, I think that could be a good time to define an `interface` for the expected JSON response. As for the no-any rule, maybe we could discuss that more offline?

Comment (ChrisAntaki) [reply src/utils/jwt.ts]: Updated to use `unknown` instead of `any`

**Pull request merged**